keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -267,59 +263,24 @@ The new allocation mode requires:
 Currently, the Service CIDRs are configured independently in each kube-apiserver using flags. During
 the bootstrap process, the apiserver uses the first IP of each range to create the special
 "kubernetes.default" Service. It also starts a reconcile loop, that synchronize the state of the
-bitmap used by the internal allocators with the assigned IPs to the Services.
+bitmap used by the internal allocators with the assigned IPs to the Services. The kubernetes.default
+Service ClusterIP is never reconciled, first apiserver creating the Service wins.
 
 With current implementation, each kube-apiserver can boot with different ranges configured.
 There is no conflict resolution, each apiserver keep writing and deleting others apiservers allocator bitmap
 and Services.
 
 In order to be completely backwards compatible, the bootstrap process will remain the same, the
 difference is that instead of creating a bitmap based on the flags, it will create a new
-ServiceCIDRConfig object from the flags (flags configuration removal is out of scope of this KEP)
+ServiceCIDR object from the flags (flags configuration removal is out of scope of this KEP)
 with a special label `networking.kubernetes.io/service-cidr-from-flags` set to `"true"`.
 
-It now has to handle the possibility of multiple ServiceCIDRConfig with the special label, and
+It now has to handle the possibility of multiple ServiceCIDR with the special label, and
 also updating the configuration, per example, from single-stack to dual-stack.
 
-The new bootstrap process will be:
+The bootstrap process will be the same, the only difference is that the apiservers will
+create the corresponding ServiceCIDRs objects from the flags.
```

Comment:
Can you expand on this - if I have 3 apiservers, each with the same flag-value, will they make 3 ranges?  If not how do they dedup?  If the flag later changes, who cleans up the mess?

The goal of the pseudocode removed here was to bootstrap the API resources from flags but not leave a ton of detritus upon upgrade.

Reply:
I have to expand here